index.html

```diff
@@ -12,7 +12,7 @@
             <div class="main__container"></div>
             <div class="main__text">
                 <h2>SCORE:</h2>
-                <div class="main__text--number">0</div>
+                <div class="text__number">0</div>
```

Comment:
you can't use `test` as you don't have `text` block anywhere. It's against BEM rules